packages/autoloader/src/autoload.php

```diff
@@ -89,6 +89,7 @@ function autoloader( $class_name ) {
 						'Automattic\Jetpack\Constants',
 						'Automattic\Jetpack\Tracking',
 						'Automattic\Jetpack\Plugin\Tracking',
+						'Automattic\Jetpack\Status',
```

Comment:
Not ideal, but there were a handful of instances where the method is called before plugins_loaded :( 

We're going to have to clean this up for all the classes sometime.